packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -48,15 +48,14 @@ $mdc-grid-list-tile-secondary-icon-size: 24px;
 }
 
 // postcss-bem-linter: define grid-list
-.mdc-grid-list {
-  @include mdc-grid-list-tile-aspect(1);
+@at-root {
   @include mdc-grid-list-tile-gutter(4px);
+  @include mdc-grid-list-tile-aspect(1);
```

Comment:
Nit: I don't think this line needs to be reordered? So the diff in this section only has to include changing `.mdc-grid-list` to `@at-root`

Reply:
Yup I will reorder